samples/DotNet/Microsoft.Azure.ServiceBus/TransactionsAndSendVia/TransactionsAndSendVia/AMQPTransactionsSendVia/Program.cs

```diff
@@ -0,0 +1,228 @@
+
+namespace TransactionsAndSendVia
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string ServiceBusConnectionString = "";
+            string QueueName = "";
+            string SendViaQueueName = "";
+            string Ex = "";
+
+            for (int i = 0; i < args.Length; i++)
+            {                
+                if (args[i] == "-ConnectionString")
+                {
+                    Console.WriteLine($"ConnectionString: {args[i + 1]}");
+                    ServiceBusConnectionString = args[i + 1]; // Alternatively enter your connection string here.
+                }
+                else if (args[i] == "-QueueName")
+                {
+                    Console.WriteLine($"Queue Name: {args[i + 1]}");
+                    QueueName = args[i + 1]; // Alternatively enter your queue name here.
+                }
+                else if (args[i] == "-SendViaQueue")
+                {
+                    Console.WriteLine($"Send Via Queue Name: {args[i + 1]}");
+                    SendViaQueueName = args[i + 1]; // Alternatively enter your queue name here.
+                }
+                else if (args[i] == "-Ex")
+                {
+                    Console.WriteLine($"Exception true / false: {args[i + 1]}");
+                    Ex = args[i + 1]; // Alternatively enter your queue name here.
+                }
+            }
+
+            var myProgram = new Program();
+
+            if (ServiceBusConnectionString != "" && QueueName != "" && SendViaQueueName == "" && Ex == "false")
+            {
+                myProgram.TransactionSample(ServiceBusConnectionString, QueueName).GetAwaiter().GetResult();
+            }
+            else if (ServiceBusConnectionString != "" && QueueName != "" && Ex == "true")
+            {
+                myProgram.TransactionError(ServiceBusConnectionString, QueueName, Ex).GetAwaiter().GetResult();
+            }
+            else if(ServiceBusConnectionString != "" && QueueName != "" && SendViaQueueName != "")
+            {
+                myProgram.TransactionSendViaSample(ServiceBusConnectionString, QueueName, SendViaQueueName).GetAwaiter().GetResult();
+            }
+            else
+            {
+                Console.WriteLine("Specify -Connectionstring and -QueueName to execute the example.");
+                Console.WriteLine("If you want to execute the Send Via sample add the parameter -SendViaQueue.");
+                Console.WriteLine("If you want to simulate an exception set -ex to \"true\".");
+                Console.ReadKey();
+            }
+
+            Console.WriteLine("Press any key to exit...");
+            Console.ReadLine();
+        }
+
+        // This sample shows a successful transaction.
+        private async Task TransactionSample(string SBCS, string QueueName)
+        {
+            // Note: A transaction cannot span more than one connection, hence you need to create your connection object
```

Comment:
This is very different from what the old client was doing. At least with SBMP.
I'd highlight it and provide some explanation.